deps/build.jl

```diff
@@ -78,4 +111,11 @@ copy_config("logo-32x32.png", juliakspec)
 copy_config("logo-64x64.png", juliakspec)
 
 eprintln("Installing julia kernelspec $spec_name")
-run(`$jupyter kernelspec install --replace --user $juliakspec`)
+run(`$jupyter-kernelspec install --replace --user $juliakspec`)
```

Comment:
Also, I don't think this will work if `$jupyter` is `ipython`, since I don't think the old IPython installed separate scripts?

Maybe the best thing is:

* Try `$jupyter kernelspec --version` and `$jupyter notebook --version` to see if they work.
* If they don't work, try `$jupyter-kernelspec` and `$jupyter-notebook`
* In `deps.jl`, define a `jupyter_notebook` constant to either `["$jupyter-notebook"]` or `[jupyter, "notebook"]`, and then run it in `notebook()` via  run(`$jupyter_notebook...`) to splat the arguments